modules/education/src/main/java/edu/stanford/spezi/modules/education/videos/EducationUiState.kt

```diff
@@ -0,0 +1,35 @@
+package edu.stanford.spezi.modules.education.videos
+
+import kotlinx.serialization.Serializable
+
+data class EducationUiState(
+    val videoSections: List<VideoSection> = emptyList(),
+)
+
+data class VideoSection(
+    val title: String,
+    val description: String,
+    val orderIndex: Int = 0,
+    val videos: List<Video> = emptyList(),
+    var isExpanded: Boolean = false,
+)
+
+@Serializable
+data class Video(
+    val title: String,
+    val description: String? = null,
```

Comment:
this can't be null anymore now, please also update the usages (reads via `description?.let`)

Reply:
Currently, the descriptions of the videos are still null. These will be added later. I have therefore temporarily left the null check in, but have already created a ticket in parallel so that we can think about revising these places later